examples/apps/presence-tracker/src/reactions.ts

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Notifications } from "@fluidframework/presence/alpha";
+import type { IPresence, ISessionClient } from "@fluidframework/presence/alpha";
+
+import type { IMousePosition, MouseTracker } from "./MouseTracker.js";
+
+/**
+ * Initializes reactions support for the app. Initialization will create a presence Notifications workspace and connect
+ * relevant event handlers. Reaction elements are added to the DOM in response to incoming notifications. These DOM
+ * elements are automatically removed after a timeout.
+ */
+export function initializeReactions(presence: IPresence, mouseTracker: MouseTracker) {
+	// Create a notifications workspace to send reactions-related notifications. This workspace will be created if it
+	// doesn't exist. We create it with no notifications. We then add the Notifications value manager. You can also
+	// initialize the workspace with value managers instead of adding them later.
+	const notificationsWorkspace = presence.getNotifications(
+		// A unique key identifying this workspace.
+		"name:reactions",
+		// Initialize an empty workspace
+		{},
+	);
+
+	// Workaround ts(2775): Assertions require every name in the call target to be declared with an explicit type annotation.
+	const notifications: typeof notificationsWorkspace = notificationsWorkspace;
+
+	// Add a Notifications value manager to the workspace.
+	notifications.add(
+		"reactions",
+		Notifications<
+			// This explicit generic type specification will not be required in the future.
+			{
+				reaction: (
+					// In the future, we'll be able to use IMousePosition here.
+					position: { x: number; y: number },
+					value: string,
+					intensity: "normal" | "intense",
```

Comment:
Do we plan to exercise the "intense" case? If not I'd say remove it to keep things simple.